src/main/java/seedu/simplykitchen/logic/commands/AddCommand.java

```diff
@@ -21,11 +22,13 @@
             + PREFIX_DESCRIPTION + "DESCRIPTION "
             + PREFIX_PRIORITY + "PRIORITY "
             + PREFIX_EXPIRY_DATE + "EXPIRY DATE "
+            + PREFIX_QUANTITY + "QUANTITY "
             + "[" + PREFIX_TAG + "TAG]...\n  "
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_DESCRIPTION + "Canned Tuna "
             + PREFIX_PRIORITY + "HIGH "
             + PREFIX_EXPIRY_DATE + "1-1-2020 "
+            + PREFIX_QUANTITY + "100 gm "
```

Comment:
A very small nitpick but "g" will be a better unit. Also, perhaps consider removing the "20g" tag in line 33?